src/components/PatientPicker.js

```diff
@@ -18,7 +20,7 @@ export const DEFAULT_PATIENT_ID = Object.keys(PATIENTS)[0];
 
 const PatientPicker = ({ loading, patientId, setPatientId }) => !loading && (
   <View style={styles.container}>
-    <View style={styles.label}><Text>Select Example Patient:</Text></View>
+    <View style={styles.label}><Text>Select synthetic patient:</Text></View>
```

Comment:
Is this the correct vernacular?

Reply:
Yes, safer to use just to prevent a potential concern.